_integration-schemas/frontapp/v1/tags_table.md

```diff
@@ -0,0 +1,74 @@
+---
+tap: "frontapp"
+version: "1"
+key: ""
+
+name: "tags_table"
+doc-link: "https://dev.frontapp.com/#analytics"
+singer-schema: "https://github.com/singer-io/tap-frontapp/blob/master/tap_frontapp/schemas/tags_table.json"
+description: |
+  The `{{ table.name }}` table contains a list of tag statistics since the last completed replication job through the most recent iteration of the defined [**Incremental Range**](#add-stitch-data-source) (day or hour).
+
+  This table will include tags in your {{ integration.display_name }} account.
+
+  **Note**: During the historical replication job, all increments since the **Start Date** will be replicated. This will result in the first record for this table being an aggregated record across all tags.
+
+replication-method: "Key-based Incremental"
```

Comment:
This table needs a Replication Key. How do the other tables in this integration replicate? What about the table that's already documented (`teams_table`) - can that give you any clues?